broker/src/test/java/io/zeebe/broker/BrokerShutdownTest.java

```diff
@@ -23,20 +21,13 @@
 import java.time.Duration;
 import org.junit.Rule;
 import org.junit.Test;
-import org.junit.rules.RuleChain;
-import org.junit.rules.Timeout;
 
 public class BrokerShutdownTest {
```

Comment:
Side note, does this test make sense as a QA test? It just checks that it releases the sockets once shut down...couldn't that just go into the broker test suite?

Reply:
It is in the broker test suite :sweat_smile: before it was in the QA module ;